client/tests/integration/asset.rs

```diff
@@ -296,7 +296,7 @@ fn find_rate_and_make_exchange_isi_should_succeed() {
     let alice_id = ALICE_ID.clone();
     let alice_can_transfer_asset = |asset_id: AssetId, owner_key_pair: KeyPair| {
         let instruction = Grant::permission(
-            PermissionToken::new(
+            Permission::new(
```

Comment:
please open a separate PR for this change. We'll merge it and then you can rebase this PR on top of it

Reply:
Opened #4635, will rebase on top of it